src/peergos.css

```diff
@@ -97,6 +97,27 @@ table {
                                   not supported by any browser */
 }
 
+.feedback-button{
+    height:40px; 
```

Comment:
it looks like all this new css isn't used either so presumably it can be removed?

Reply:
It's used on line 108 of `filesystem.html`.